Production/crab/configs/2018/Embedded.txt

```diff
@@ -0,0 +1,6 @@
+sampleType=MC_18
```

Comment:
- sampleType for emb is different: Emb_18ABC or Emb_18D depending on era
- add `requireGenMatch=False`

Reply:
@kandrosov Have to raise this comment again because requiring `requireGenMatch=False` is inconsistent with the following definition of `GenMatchToTauType` function:
https://github.com/cms-tau-pog/TauMLTools/blob/514fcbb2cedbed0d07c018b042741a5a67b39ec3/Analysis/interface/AnalysisTypes.h#L64-L80
i.e: when having gen_match == GenLeptonMatch::NoMatch and sample_type == SampleType::Embedded this functon throws exeption. What will be the procedure of using the Embedded events that does not represent events with taus?

Reply:
You are right, the definition of `TauType` is not up-to-date with the latest production and should be updated. We need to introduce: `TauType::tau_emb` and `TauType::jet_emb` instead of single `TauType::emb`.